### PR TITLE
Release 0.4.0

### LINKED ISSUE #174: Prepare for release `0.4.0`
## Depends on tasks:
- #9 
- #59 
- #140

## Description:
Prepare the documentation and source for release.

## User requirements:
User should be able to go to the release page in repo, get the information about changes that were done and be able to download/setup the app

## Dev requirements and restrictions:

- Create a new branch for the release with the name "release-0.4.0" and set it up for release;
- Create a new branch to bump the version, if requires.
- Check the CHANGELOG.md, verify that there are no missed changes, and update with release information;
- Add the necessary info to the [Release_notes](https://github.com/ios-course/ironfoudation-team-project/blob/main/release_notes.md) in a user-friendly format based on CHANGELOG.md.
- Before merging to the main branch set up the tag "v.0.4.0" on the last commit before the merge commit.
- Publish the Release under the Release [section on GitHub.](https://github.com/ios-course/ironfoudation-team-project/releases)

### PR DESCRIPTION
**Pull request by kirpichonok**
## Summary
This PR is a part of the work to resolve #174.

## Changes
- Update CHANGELOG.md with the info related to release 0.4.0.
- Update `release_notes` file with the info related to release 0.4.0.

## Checks to complete
- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] I've updated `CHANGELOG.md` file.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: release_notes.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ The file describes all the changes, that are affected the usage of the applicati
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
 
+## 0.4.0
+
+### New features
+
+`User profile`: Add the ability for sitters and owners to enter information about themselves. The entered information is saved after the app is closed.
+ 
+`Connect pet sitters and pet owners`: Add a connection to the remote server what enables pet owners to have a list of registered pet sitters.
```
Comment (NadzeyaShpakouskaya) [inline release_notes.md:15]:
```suggestion
`Find your sitter`: If you have a furry friend, you can easily browse through our list of  pet sitters to find the one that best suits your needs.
```

File: release_notes.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ The file describes all the changes, that are affected the usage of the applicati
 
 The format is based on [Release_notes format](https://github.com/ios-course/ironfoudation-team-project/wiki/Release_notes-format).
 
+## 0.4.0
+
+### New features
+
+`User profile`: Add the ability for sitters and owners to enter information about themselves. The entered information is saved after the app is closed.
```
Comment (NadzeyaShpakouskaya) [inline release_notes.md:13]:
```suggestion

`Create a Sitter Profile`: Join us as a sitter and start earning money with your pet care skills!
```

**Review by NadzeyaShpakouskaya — CHANGES_REQUESTED** [review]
I think we should use a more user-friendly format in release notes.  They don't care about how our app works and what happens under the hood.

File: release_notes.md (hunk 2 above)
Comment (iBYarchitect) [inline release_notes.md:13]:
```suggestion
`User profile`: Added the possibility for the users to enter and save information about themselves on the profile screen. Data is stored persistently.
```

File: release_notes.md (hunk 1 above)
Comment (iBYarchitect) [inline release_notes.md:15]:
```suggestion
`Connect pet sitters and pet owners`: Added the possibility pet owners can see a list of pet sitters and find out additional helpful information as well as the phone number for the selected pet sitter.
```

**Review by iBYarchitect — COMMENTED** [review]
WDYT about my suggestions?

File: release_notes.md (hunk 2 above)
Comment (kirpichonok) [reply release_notes.md]: Yes, your version is obviously better.

Comment (kirpichonok) [reply release_notes.md]: As we discussed, I applied Nadia's version.

File: release_notes.md (hunk 1 above)
Comment (kirpichonok) [reply release_notes.md]: As we discussed, I applied Nadia's version.

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM

**Pull request merged**